components/page-templates/CartTemplate/CartTemplate.tsx

```diff
@@ -239,35 +283,66 @@ const CartTemplate = (props: CartTemplateProps) => {
               onFulfillmentOptionSelection={handleFulfillmentOptionSelection}
               onProductPickupLocation={handleProductPickupLocation}
             />
-            <Box py={5}>
-              <Button
-                variant="contained"
-                color="secondary"
-                onClick={handleContinueShopping}
-                startIcon={<KeyboardArrowLeft fontSize="small" sx={{ color: 'text.secondary' }} />}
-              >
-                {t('continue-shopping')}
-              </Button>
-            </Box>
           </Grid>
           {/* Order Summary */}
-          <Grid item xs={12} md={4} sx={{ paddingRight: { xs: 0, md: 2 } }}>
+          <Grid
+            item
+            xs={12}
+            md={4}
+            sx={{ paddingRight: { xs: 0, md: 2 }, alignContent: 'pull-right' }}
+          >
+            <PromoCodeBadge
+              onApplyCouponCode={handleApplyPromoCode}
+              onRemoveCouponCode={handleRemovePromoCode}
+              promoList={cart?.couponCodes as string[]}
+              promoError={!!promoError}
+              helpText={promoError}
+            />
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            md={4}
+            sx={{ paddingRight: { xs: 0, md: 2 }, alignContent: 'pull-right' }}
+          ></Grid>
+          <Grid
+            item
+            xs={12}
+            md={4}
+            sx={{ paddingRight: { xs: 0, md: 2 }, alignContent: 'pull-right' }}
+          >
             <OrderSummary {...orderSummaryArgs}>
               <Stack direction="column" gap={2}>
-                <LoadingButton
+                {/* <LoadingButton
```

Comment:
Remove commented code or provide a comment with reason for commenting this code.

Reply:
Removed commented codes